### DISCUSSION TITLE
No log and HA being unresponsive when Cloudlared add-on started

### DISCUSSION THREAD

**Discussion by FrozenBull** [homeassistant-apps/app-cloudflared#285] [Q&A] [answered]
Hi All,

I have been trying to set-up a Cloudlare access to my local HA and I have followed this tutorial (https://www.linkedin.com/pulse/what-do-domain-name-how-get-one-free-tobias-brenner?trk=public_post-content_share-article) to set-up my free domain and Cloudflare. 
I have installed Cloudflared add-on, entered my domain name in the Config tab (basically following this video from Everything Smart Home - https://www.youtube.com/watch?v=xXAwT9N-7Hw), run the add-on, but I do not get any logs - it just stays empty, even after hitting refresh several times, and my HA instance is becoming very unresponsive - trying to load a dashboard and enter setting takes forever - even trying to stop the add-on takes at least 5 minutes to complete.
When the add-on is turned-off, everything works fine with no issues.

Has any of you had a similar case and could share a solution?
Do you have any hints?

If you need me to post more information, let me know.

Thanks in advance for all your help.

Aleksander

**Comment by brenner-tobias** [comment, marked as answer]
@FrozenBull,
thanks for raising this. It is the first time I hear of such an issue. I suggest you uninstall the add-on and follow the documentation step-by-step. If an issue persists, please raise an issue in GitHub with as many logs and details as possible.

I hope this helps.
